wdl2cwl/tests/cwl_files/smoove.cwl

```diff
@@ -0,0 +1,24 @@
+class: CommandLineTool
+id: Call
+inputs:
+- id: bamFile
+  type: File
+- id: bamIndex
+  type: File
+- id: referenceFasta
+  type: File
+- id: referenceFastaFai
+  type: File
+- id: sample
+  type: string
+requirements:
+- class: InitialWorkDirRequirement
+  listing:
+  - entryname: example.sh
+    entry: 'set -e mkdir -p $(inputs.outputDir) smoove call  --outdir $(inputs.outputDir)  --name
```

Comment:
This should have been broken up into three separate lines, not smooshed all together 😃

Reply:
Let's do a video chat and we can debug this together